rswag-specs/lib/rswag/specs/swagger_formatter.rb

```diff
@@ -126,19 +126,31 @@ def upgrade_response_produces!(swagger_doc, metadata)
         target_node = metadata[:response]
         upgrade_content!(mime_list, target_node)
         metadata[:response].delete(:schema)
+        metadata[:response].delete(:examples)
```

Comment:
curious why is this?

Reply:
The library is making mutative changes to response object, and that is put into the responses section of the swagger doc. If you don't delete the upgraded key here, it would end up in the the file output in the wrong spot. This is not the way I would design it if i were writing this from scratch but I'm following the style of the code as it is.